external/jenkins/test.sh

```diff
@@ -16,7 +16,7 @@ source $(dirname "$0")/test_base_functions.sh
 # Set up Java to be used by the jenkins test
 echo_setup
 
-export JAVA_TOOL_OPTIONS="-Dfile.encoding=UTF8"
+export JAVA_TOOL_OPTIONS="$JAVA_TOOL_OPTIONS -Dfile.encoding=UTF8"
```

Comment:
This is defined specifically for jenkins. No need to do the prefix.

Reply:
not true, as stated above, or I'm missing something.  java_tool_options is widely used JDK's variable.